src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Proxy agent chunk requests and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code. (Default: js-agent.newrelic.com CDN)
+2. Where to send analytics. (Default: bam.nr-data.net)
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its visitor analytics through its `nrbeacon.yoursite.com` proxy which would not be blocked by anti-adware being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-and-paste _or_ NPM installation methods. There is no UI configuration at this time, and APM auto-injected agents are not able to use this until further improvements are made.
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch chunks or send data.
+
+For example, using copy and paste:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-and-paste installation to fetch agent chunks. This implies you are hosting the file somewhere instead it being retrieved via our CDN. 
```

Comment:
This is not accurate. The customer's are not hosting the files, they are only providing a reverse proxy where our CDN is the upstream origin.